### ISSUE TITLE
License + Copyright clarifications

### ISSUE THREAD

**Issue by sypets** [TYPO3-Documentation/T3DocTeam#96]
# Problems 

* we have a mixture of "Open Publication" (for older docs) + CC (for newer docs) in documentation
* we have mixture of CC: CC-BY + CC-BY-SA-NC
* we use Source Snippets from extensions + TYPO3 source (which are GPL) verbatim in our docs 
  * update: this should not be a problem
* we want to move things from the Wiki. The Wiki uses open content license
* the Extbase / Fluid book was never assigned a license. It was originally just a translation by a group of authors from the community. Sebastian Kurfürst has agreed to switching to commonly used CC-BY-SA-NC. But a number of people contributed to this project as well. How do we handle this?
* CC-BY etc. mean **attributing** the **author** of the original work (if the work is used elsewhere). We have multiple authors. How should they be attributed?
* we have a copyright notice in the footer. What exactly does that mean? Should we still list "Documentation Team" or "Association" or "Core Team" even if this does not really correctly credit the authors (which are often more people)

![copyright](https://user-images.githubusercontent.com/13206455/61180785-54972f00-a61c-11e9-9944-d150009a7c28.png)

Mixed licenses (which we currently have) may be a problem, especially if things are moved from one manual to another which is often the case. 

# Questions

* Is CC-BY-SA-NC the best fit here? Or CC-BY-SA? CC-BY? Or ...
* Can we (safely) switch from Open Publication to CC...
* Can we (safely) switch from no license to CC-... for Extbase / Fluid book?
* can we safely move things from the wiki to other docs effectively changing the license?
* the Changelogs + sysext docs are in the core. Are they automatically GPL? What happens if a Changelog is copied to the main doc or uses as basis for main doc? (GPL and CC with NC are incompatible - see below)
* **BY** means attribution. How should authors be attributed? If someone creates something based on our work, this is important. See https://wiki.creativecommons.org/wiki/best_practices_for_attribution for some examples
* who should be listed as authors? We currently have "Documentation Team", "Association", individual authors or "TYPO3 community"
  * in copyright notice in footer
  * on start page
* should all authors be listed (e.g. in credits)? see also #30 

# Information:

**About licenses in our docs**

* Wiki: [open content license](https://wiki.typo3.org/TYPO3Wiki:About):

> Please note that all contributions to the TYPO3 wiki are considered to be released under the Open Content License. If you don't want your writing to be edited mercilessly and redistributed at will, then don't submit it here.

* older manuals: Open Publication License
* newer manuals: [CC-BY-SA-NC](https://creativecommons.org/licenses/by-nc-sa/4.0/)

**about CC**

* **CC**=Creative-Commons
* **BY** : attribution

>  You must give appropriate credit, provide a link to the license, and indicate if changes were made. 

* **SA**: share alike: use same licence if basing your work and sharing it

>  If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original.

> The ShareAlike licenses are known as "copyleft" licenses: they use copyright to ensure that the freedoms associated with a licensed work survive as the work is modified and adapted by others, and that those freedoms attach to adaptations of the work as well.

https://wiki.creativecommons.org/wiki/ShareAlike_interpretation

* **NC**: non commercial

from least restrictive to more:

BY -> CC-BY-SA -> CC BY-SA-NC

* **ND** no derivatives: work based on original with changes is not allowed

---
**About compatibility:**

>  the GNU GPL license and any of the Creative Commons Non-Commercial licenses are incompatible.

https://opensource.stackexchange.com/a/5621/15477

> the Open Publication License is not a share-alike license

> According to the Free Software Foundation, the Open Publication License "can be used as a free documentation license" and is "a copyleft free documentation license provided the copyright holder does not exercise any of the 'LICENSE OPTIONS' listed in Section VI of the license.

https://en.wikipedia.org/wiki/Open_Publication_License

----

About **attribution**: Most examples recommend to list the **title**, **author**, link to **original work** and link to **license**, e.g. see https://wiki.creativecommons.org/wiki/Best_practices_for_attribution

# Links to resources

* [CC-BY-NC-SA](https://creativecommons.org/licenses/by-nc-sa/2.0/)
* Open Publication license: http://www.opencontent.org/openpub/ (is generally considered as outdated and usage discouraged, e.g. by original authors)
* List of documentation licenses on gnu.org: https://www.gnu.org/licenses/license-list.en.html#FreeDocumentationLicenses
* TYPO3 wiki license info: https://wiki.typo3.org/TYPO3Wiki:About
* Information about licenses in "Writing Documentation": https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/GeneralConventions/Licenses.html

# Related:

* give credit, acknowledgement #95, #30

**Comment by sypets** [comment]
Update:

* see open issue in "Licensing Team" (forge): [Clarify possible licenses for official TYPO3 documentation](https://forge.typo3.org/issues/46163)

**Comment by alexander-nitsche** [comment]
Some decisions have been made in this regard:
1. The common documentation license is CC-BY.
2. Documentations licensed under "Open Publication" remain that for the time being.
3. The wiki does not exist anymore and we transferred all useful information into the existing documentation, without changing the license of the target documentation.
4. We assigned the new default license CC-BY to the Extbase book.
5. Attributing the authors of the original work when reusing the documentation licensed under CC-BY should be done in a best-effort way. naming the original URL of the excerpt is probably fine, e.g.
```
"TYPO3 CMS is an Open Source Enterprise Content Management System with a large global community, backed by the approximately 900 members of the TYPO3 Association." [1]

[1] See https://typo3.org/.
```
6. The copyright claim is not legally binding, but it should describe all contributors of the documentation. So "Copyright since 1998 by the TYPO3 Documentation Team" is wrong, as there are many contributors outside the team. We currently use the claim "Copyright since 1998 by the TYPO3 contributors" or "Copyright since 1998 by <vendor> & contributors", where <vendor> can be an excelling author or a company etc.
7. Code license is not to be mixed with documentation license. Documentation has always to define an own license. Thus the TYPO3 code mostly be licensed under GPL2/3 does not have influence on the license of the documentation of the same extension. We usually license the documentation of the TYPO3 system extensions under OPL (old) and CC-BY (new). Of course there are some exceptions:)
